src/auth/token.vue

```diff
@@ -61,13 +61,17 @@ export default {
 	},
 	created: function() {
 		if (this.$route.query.missingcsc) {
-			this.$router.replace({name: 'home', params: {missingCsc: true}})
+			this.$auth.setLoginError("missingcsc")
+			this.$auth.setUser(null)
```

Comment:
should both:
		this.$auth.setLoginError("missingcsc")
 			this.$auth.setUser(null)
be capsulated into a single function inside this.$auth.setLoginError("missingcsc", null) and then in setLoginError would check if that second one is defined.

```
fuu = function(a, b) {
  if (b) {
    console.log("b")
  }
  console.log("a")
}

fuu("fuu")
VM236:5 a
undefined
fuu("fuu", "bar")
VM236:3 b
VM236:5 a
undefined
```